packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -109,53 +78,26 @@ Template.AssistifyCreateRequest.events({
 		const position = input.selectionEnd || input.selectionStart;
 		const length = input.value.length;
 		document.activeElement === input && e && /input/i.test(e.type) && (input.selectionEnd = position + input.value.length - length);
-		if (input.value) {
-			t.delaySetExpertise(input.value);
-			t.checkExpertise(input.value);
-		} else {
-			t.delaySetExpertise(false);
-			t.expertise.set('');
-		}
-		if (t.expertise.get() && t.requestTitle.get()) {
-			const requestName = `${ t.expertise.get() }-${ t.requestTitle.get() }`;
-			t.invalidTitle.set(!validateRequestName(requestName));
-			t.requestTitleInUse.set(undefined);
-			t.checkRequestName(requestName);
-		}
+		t.expertise.set(input.value);
```

Comment:
The copying of the actual value should happen immediately in order to have the value at hand for subsequent calls